src/shared/components/palette/complex/read-complex-field-new-table.component.ts

```diff
@@ -0,0 +1,64 @@
+import { Component, OnInit } from '@angular/core';
+import { AbstractFieldReadComponent } from '../base-field/abstract-field-read.component';
+
+@Component({
+  selector: 'ccd-read-complex-field-new-table',
+  templateUrl: './read-complex-field-new-table.html',
+  styleUrls: ['./read-complex-field-new-table.scss']
+})
+export class ReadComplexFieldNewTableComponent extends AbstractFieldReadComponent implements OnInit {
```

Comment:
Not sure I like the ...NewTable... in the name. Can we think of another name?

Reply:
Yeah, you are right there, I will come with some better name

Reply:
changed the name